Cargo.toml

```diff
@@ -23,25 +23,25 @@ native-tls = ["hyper-tls"]
 rustls-tls = ["hyper-rustls"]
 
 [dependencies]
-webdriver = { version = "0.43.0", default-features = false }
+webdriver = { version = "0.44", default-features = false }
 url = "2.2.2"
-serde = { version = "1.0", features = [ "derive" ] }
+serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0"
-futures-core = "0.3.0"
-futures-util = "0.3.0"
-tokio = { version = "1", features = [ "sync", "rt" ] }
-hyper = { version = "0.14", features = [ "stream", "client", "http1", "http2" ] }
-cookie = { version = "0.14", features = ["percent-encode"] }
+futures-core = "0.3"
+futures-util = "0.3"
+tokio = { version = "1.0", features = ["sync", "rt"] }
+hyper = { version = "0.14", features = ["stream", "client", "http1", "http2"] }
+cookie = { version = "0.16.0-rc.1", features = ["percent-encode"] }
```

Comment:
This is a very unfortunate dependency given that we expose types from `cookie` in the public API, and so when we switch to the non-rc version, that'll arguably be a breaking change. I may end up releasing this change as an `rc` as well until `cookie` has a full release.